wxfs/indexer/file_parsing.py

```diff
@@ -11,7 +15,15 @@ def get_wx_file_info(wx_file):
     is externally determined or determined from its filename.
     """
     line = wx_file.readline()
-    location_part, morphed, file_version, creation_date_part = line.split(" | ")
+    try:
+        location_part, morphed, file_version, creation_date_part = line.split(" | ")
+    except ValueError as e:
+        logger.error(
+            "Could not split the following line by ' | '... will have to return None: '%s'",
+            line,
+        )
+        return None, None
+
```

Comment:
Just to check: It's 4 values guaranteed/required, no other cases?

Reply:
Yeah, good question.

```
   try:
        location_part, morphed, file_version, creation_date_part = line.split(" | ")
  ```

I see, `location_part` and `create_date_part` used below as mandatory fields. The two middle metadata item aren't used at all. They could be omitted as variable names, but it does help with readability a bit. But, we'd have to get more creative or forgiving to be able to handle files that didn't have the additional metadata.

Reply:
Yes, that's why I named them like that. But we could just do something like

```
parts = line.split(" | ")
try:
    location_part = parts[0]
except:
    # ...

try:
    creation_date_part = parts[3]
except:
    # ...
```

Reply:
Sure, I agree that it would theoretically be valuable to isolate those two pieces of information. But as it stands, we can't do anything without both location and creation_date. I'll leave this as it is in this PR, but log an issue to have `creation_date` fall back to the filesystem datastamp if it's not available in the file metadata.